### ISSUE TITLE
(Angular 13) Applies to Formly From - Margin top for address fields are bigger than it used to be

### ISSUE THREAD

**Issue by wonko-gsa** [GSA/sam-styles#581]
1. We've updated to **Angular 13** and **@gsa-sam packages** as below: 

```
    "@gsa-sam/components": "13.0.0-beta.3",
    "@gsa-sam/layout": "13.0.0-beta.1",
    "@gsa-sam/layouts": "13.0.0-beta.0",
    "@gsa-sam/ngx-uswds": "13.0.0-beta.1",
    "@gsa-sam/ngx-uswds-icons": "13.0.0-beta.2",
    "@gsa-sam/sam-formly": "13.0.0-beta.3",
    "@gsa-sam/sam-material-extensions": "13.0.0-beta.3",
    "@gsa-sam/sam-styles": "0.0.145",
    "@gsa-sam/shared": "13.0.0-beta.1",
```

2. Found that the spaces between address fields are bigger than they are used to be. 
* Notes: The right side is COMP **without** the updates, and the left side is localhost **with** the updates in the screenshot.

a,
![11  margin-exclusion-SED-address-fields ](https://user-images.githubusercontent.com/117218338/199347719-446309a9-0092-45ef-8840-704482f4d503.png)

b, 
![23  margin-non-fed-entities-validate-entities-](https://user-images.githubusercontent.com/117218338/199347728-de46e5bb-cf53-496a-8161-72f7e98b11c4.png)

3. It seems those changes are coming from updating CSS styles below.

![11  css-style-minc-vs-comp-margin-exclusion-SED-address-fields ](https://user-images.githubusercontent.com/117218338/199350306-cd7964d8-f1c0-40ad-a3c8-45791058e1b4.png)
* Notes: The right side is COMP with the updates, and the left side is MINC without the updates in the screenshot.

```
formly-field {
    display: block;
    margin-top: 1.5rem;
}
```

4. Here is a link of code for formly config (line 136 - line392)
https://github.helix.gsa.gov/GSA-IAE-APPS/sam-frontend-entity/blob/integration-42.3/apps/entity-tier2/src/app/get-entity-id-V2/entity-form-V2/entity-form-V2.component.ts#L136-L392

**Comment by yerramshilpa** [comment]
Issues resolved with latest versions of sds 

@gsa-sam/components@13.0.1
@gsa-sam/sam-formly@13.0.1
@gsa-sam/sam-styles@0.0.147

**Comment by yerramshilpa** [comment]
issue has been resolved in @gsa-sam/sam-formly@13.0.2 version
